### DISCUSSION TITLE
How to read JSONDocument of gltf files with WebIO?

### DISCUSSION THREAD

**Discussion by neciszhang** [donmccurdy/glTF-Transform#1106] [Q&A] [answered]
Hi, 
I want to read JSONDocument gltf file(bin+gltf+textures) with WebIO, how to do with WebIO?
Thanks.

**Comment by donmccurdy** [comment, marked as answer]
Please share more detail about what you're trying to do, and any issues you're having. Are you having trouble constructing a JSONDocument, and if so from what data? If the glTF files are at a remote URL you probably don't need to construct the JSONDocument structure first, instead call `await io.read(url)` with that URL.

Related:

- https://gltf-transform.dev/modules/core/classes/WebIO
- https://gltf-transform.dev/modules/core/interfaces/JSONDocument

**Reply by neciszhang** [reply]
Thank you for your quick reply. I found the historical answer. [This is what I need.](https://github.com/donmccurdy/glTF-Transform-View/blob/52431daf421aab23a426eabad8e86dbbcab25dca/examples/dropzone.ts#L45-L65)
